src/platform/forms-system/src/js/web-component-patterns/radioOptionalPattern.jsx

```diff
@@ -0,0 +1,79 @@
+import VaRadioField from '../web-component-fields/VaRadioField';
+
+/**
+ * Web component v3 uiSchema for generic optional radio field
+ *
+ * Usage uiSchema:
+ * ```js
+ * exampleRadio: radioOptionalUI({
+ *  title: 'Select animal',
+ *  labels: {
+ *      dog: 'Dog',
+ *      cat: 'Cat',
+ *      octopus: 'Octopus',
+ *  },
+ *  hint: 'This is a hint',
+ * })
+ * ```
+ *
+ * Advanced labels:
+ * ```js
+ * labels: {
+ *    dog: 'Dog',
+ *    cat: 'Cat',
+ * },
+ * descriptions: {
+ *    dog: 'This is a dog',
+ *    cat: 'This is a cat',
+ * }
+ * ```
+ *
+ * Usage schema:
+ * ```js
+ * exampleRadio: radioOptionalSchema()
+ * ```
+ * @param {UIOptions & {
+ *  title?: UISchemaOptions['ui:title'],
+ *  labels?: UISchemaOptions['ui:options']['labels'],
+ *  description?: UISchemaOptions['ui:description'],
+ *  labelHeaderLevel?: UISchemaOptions['ui:options']['labelHeaderLevel'],
+ *  hint?: string,
+ * }} options
+ * @returns {UISchemaOptions}
+ */
+export const radioOptionalUI = ({
+  title,
+  description,
+  errorMessages, // ignored
+  required, // ignored
+  ...uiOptions
+}) => {
+  if (required || errorMessages) {
+    // eslint-disable-next-line no-console
```

Comment:
@rhasselle-oddball I added a console warning, but wasn't sure if it was necessary? WDYT?

Reply:
if we keep this, probably throw a proper error if they try required since it doesn't make sense. (I think we do something similarly in checkboxgroup where 'required' IS required, so the opposite.
 errorMessages can still happen for other reasons than required though, I think like relationship to veteran has that pattern?  so maybe no error or warning for that.